src/TulsiGenerator/DeploymentTarget.swift

```diff
@@ -39,7 +39,7 @@ public enum CPU: String {
   }
 
   var watchCPU: CPU {
-    return isARM ? .armv7k : .i386
+    return isARM ? .armv7k : .x86_64
```

Comment:
I forgot to update: This change broke the builds of the generated project if you use Xcode 11.x to build. Internally we have a compiler version check here to work around that since we use the same Xcode version to build Tulsi and our app.

Reply:
11.x as opposed to? I guess I'll revert

Reply:
to 12.x

Reply:
I reverted. If you want you can submit a PR that's compatible with both

Reply:
https://github.com/bazelbuild/tulsi/pull/162